include/mbedtls/mbedtls_config.h

```diff
@@ -1541,6 +1541,15 @@
  */
 //#define MBEDTLS_SSL_TLS1_3_COMPATIBILITY_MODE
 
+/**
+ * \def MBEDTLS_SSL_TICKET_NONCE_LENGTH
+ *
+ * Size in bytes of a ticket nonce. This is not used in TLS 1.2.
+ *
+ * This must be smaller or equal to 255.
```

Comment:
`smaller` is the wrong word here. Any of the following are acceptable:

`This must be less than or equal to 255.`

`This must be less than 256.`

`This must be <= 255.`

`This must be < 256.`

And surely a corresponding check in `check_config.h`

```
#if defined(MBEDTLS_SSL_TICKET_NONCE_LENGTH) && MBEDTLS_SSL_TICKET_NONCE_LENGTH >= 256
#error "MBEDTLS_SSL_TICKET_NONCE_LENGTH must be less than 256"
#endif
```